packages/snaps-execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -162,7 +164,7 @@ export class BaseSnapExecutor {
         try {
           return getSafeJson(result);
         } catch (error) {
-          throw new TypeError(
+          throw ethErrors.rpc.invalidInput(
```

Comment:
Technically this is invalid output right? Not sure which RPC error makes the most sense

Reply:
Yeah, I wasn't sure either. There is no code for "invalid output," so I was thinking it's invalid input for the response? 😅

Reply:
Maybe `ethErrors.rpc.internal` instead?